src/libtls/tls_peer.c

```diff
@@ -1217,6 +1217,7 @@ static status_t send_client_hello(private_tls_peer_t *this,
 	}
 	rng->destroy(rng);
 
+	count = this->crypto->get_cipher_suites(this->crypto, &suites);
```

Comment:
Add a comment and perhaps a newline.

```suggestion
	/* determine supported suites before the versions as they might change */
	count = this->crypto->get_cipher_suites(this->crypto, &suites);
	
```

Reply:
I agree.